imesec-full-stack.yml

```diff
@@ -0,0 +1,91 @@
+version: "3.4"
+services:
+  lh-web:
+    image: razgrizone/lightning-hacks-website
+    environment:
+      VIRTUAL_HOST: lh.torradeira.imesec.ime.usp.br
+    deploy:
+      replicas: 2
+      restart_policy:
+        condition: any
+        delay: "0"
+    ports:
+      - "8994:5000"
+
+  lh-api:
+    image: razgrizone/lightning-hacks-api
+    environment:
+      VIRTUAL_HOST: api.lh.torradeira.imesec.ime.usp.br
+    deploy:
+      replicas: 2
+      restart_policy:
+        condition: any
+        delay: "0"
+    ports:
+      - "8992:5000"
+    volumes:
+      - /tokens/lh-api.pickle:/data/token.pickle
+

```

Comment:
You said you'd need to put all the images in the same network, however I'm not seeing the network over here. You need to attach the network to each deployment, isn't it?